bigchaindb/pipelines/block.py

```diff
@@ -111,28 +112,27 @@ def write(self, block):
         """Write the block to the Database.
 
         Args:
-            block (dict): the block of transactions to write to the database.
+            block (Block): the block of transactions to write to the database.
 
         Returns:
             The block.
         """
-        logger.info('Write new block %s with %s transactions',
-                    block['id'],
-                    len(block['block']['transactions']))
+        logger.info('Write new block {} with {} transactions'.format(block.id,
+                    len(block.transactions)))
         self.bigchain.write_block(block)
         return block
 
     def delete_tx(self, block):
         """Delete transactions.
 
         Args:
-            block (dict): the block containg the transactions to delete.
+            block (Block): the block containg the transactions to delete.
 
         Returns:
             The block.
```

Comment:
@sohkai from the code, you can see it returns the block, that it received as an argument